src/components/ApplicationGroup/CreateAppGroup.tsx

```diff
@@ -192,6 +192,14 @@ export default function CreateAppGroup({ appList, selectedAppGroup, closePopup }
         )
     }
 
+    const nameErrorMessage = (): string => {
+        if (!appGroupName) {
```

Comment:
Also will this get executed when `appGroupName` is not present because we already have optional length check on line 223